app/src/org/commcare/adapters/EntityListAdapter.java

```diff
@@ -96,9 +105,6 @@ public EntityListAdapter(Activity activity, Detail detail,
             if (sort.length != 0) {
                 sort(sort);
             }
-            filterValues("");
-        } else {
-            setCurrent(new ArrayList<>(full));
```

Comment:
It didn't make sense to me to do a `filterValues("");`, so I removed that call and made both sync and async loader modes call `setCurrent(new ArrayList<>(full));` (see last line of constructor).